components/content_settings/core/browser/content_settings_util.cc

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/content_settings/core/browser/content_settings_util.h"
+
+#include "brave/components/brave_shields/common/brave_shield_constants.h"
+#include "components/content_settings/core/browser/host_content_settings_map.h"
+#include "components/content_settings/core/common/content_settings_types.h"
+// #include "components/content_settings/core/common/content_settings_utils.h"
```

Comment:
erase

Reply:
missed this, but I'll get it on the follow-up network service PR